.gitignore

```diff
@@ -128,5 +128,9 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
+# Vagrant
```

Comment:
@ShellyGolden just remove the comment, because it doesn't explain anything more then what we can include from the line below it :)